RELEASE/scripts/autoscend/auto_quest_level_8.ash

```diff
@@ -37,30 +248,12 @@ boolean L8_trapperGround()
 		{
 			auto_sourceTerminalEducate($skill[Extract], $skill[Duplicate]);
 		}
-		autoAdv(1, $location[The Goatlet]);
+		autoAdv($location[The Goatlet]);
 		auto_sourceTerminalEducate($skill[Extract], $skill[Portscan]);
 		return true;
 	}
 
-	if(item_amount(oreGoal) >= 3)
-	{
-		if(item_amount($item[Goat Cheese]) >= 3)
-		{
-			auto_log_info("Giving Trapper goat cheese and " + oreGoal, "blue");
-			visit_url("place.php?whichplace=mclargehuge&action=trappercabin");
-			return true;
-		}
-		auto_log_info("Yay for goat cheese!", "blue");
-		handleFamiliar("item");
-		if(get_property("_sourceTerminalDuplicateUses").to_int() == 0)
-		{
-			auto_sourceTerminalEducate($skill[Extract], $skill[Duplicate]);
-		}
-		autoAdv(1, $location[The Goatlet]);
-		auto_sourceTerminalEducate($skill[Extract], $skill[Portscan]);
-		return true;
-	}
-	else if((my_rain() > 50) && (have_effect($effect[Ultrahydrated]) == 0) && (auto_my_path() == "Heavy Rains") && have_skill($skill[Rain Man]))
+	if((my_rain() > 50) && (have_effect($effect[Ultrahydrated]) == 0) && (auto_my_path() == "Heavy Rains") && have_skill($skill[Rain Man]))
```

Comment:
redundant checks. it is impossible to have more than 0 rain if your path is not heavy rains. it is impossibly to have rain man skill without being in path heavy rains.
Also I somewhat disagree about the notion of wasting copiers just because you have ultrahydrated. Seems like bad strategic decision to me.

.. and I just realized git is incorrectly saying you changed it. when in fact this was how it was done all along. Actually also, isn't there a heavy rains utility function where you just send it the monster name you want to copy and it checks all those things for you? IIRC it also already summons mountain men there too...

I will take a look at it in more depth later after you commit this.

Reply:
technically I did change it, it was an `else if` now because I removed the `if` above it, it's an `if` but I haven't changed the condition at all as you saw.